train.py

```diff
@@ -27,7 +27,7 @@
 WAVENET_PARAMS = './wavenet_params.json'
 STARTED_DATESTRING = "{0:%Y-%m-%dT%H-%M-%S}".format(datetime.now())
 SAMPLE_SIZE = 100000
-
+L2_REGULARIZATION_STRENGTH=0.
```

Comment:
> I like the idea of a second JSON file.

I dislike the idea of a second JSON file.

> a second JSON file with training parameters

Weren't you the one complaining about too many hyperparameters? Well I'm going to complain about too many config files. How about just putting two objects within a single file, one for model parameters and a second for training params.